docs/index.md

```diff
@@ -25,50 +25,60 @@ under the License.
 
 This documentation is for Apache Flink version {{ site.version_title }}. These pages were built at: {% build_time %}.
 
-Apache Flink is an open source platform for distributed stream and batch data processing. Flink’s core is a streaming dataflow engine that provides data distribution, communication, and fault tolerance for distributed computations over data streams. Flink builds batch processing on top of the streaming engine, overlaying native iteration support, managed memory, and program optimization.
+<div class="row">
+<div class="col-sm-6" markdown="1">
 
-## First Steps
+### Try Flink
 
-* **Code Walkthroughs**: Follow step-by-step guides and implement a simple application or query in one of Flink's APIs. 
-  * [Implement a DataStream application](./getting-started/walkthroughs/datastream_api.html)
-  * [Write a Table API query](./getting-started/walkthroughs/table_api.html)
+Apache Flink is an open source platform with expressive APIs for defining stream and batch data processing jobs, and a robust and scalable engine for executing those jobs.
 
-* **Docker Playgrounds**: Set up a sandboxed Flink environment in just a few minutes to explore and play with Flink.
-  * [Run and manage Flink streaming applications](./getting-started/docker-playgrounds/flink-operations-playground.html)
+To quickly get a general understanding of how Flink works and what it offers for your use case, work your way through one of these tutorials:
 
-* **Concepts**: Learn about Flink's concepts to better understand the documentation.
-  * [Stateful Stream Processing](concepts/stateful-stream-processing.html)
-  * [Timely Stream Processing](concepts/timely-stream-processing.html)
-  * [Flink Architecture](concepts/flink-architecture.html)
-  * [Glossary](concepts/glossary.html)
+* [Fraud Detection with the DataStream API]({% link try-flink/datastream_api.md %})
+* [Real Time Reporting with the Table API]({% link try-flink/table_api.md %})
+* [Intro to the Python Table API]({% link try-flink/python_table_api.md %})
+* [Flink Operations Playground]({% link try-flink/flink-operations-playground.md %})
 
-## API References
+### Learn Flink
```

Comment:
What about making this the top-most item (also in the navigation), and renaming it to something like "Flink Overview"/"Flink 101" (can't really think of anything better right now 😬 )? This content kind of precedes the "Getting Started" material — it's more of a...basic foundation?